nx/test/nx_test.exs

```diff
@@ -1628,4 +1628,51 @@ defmodule NxTest do
                )
     end
   end
+
+  describe "sigil" do
+    test "evaluates to tensor" do
+      import Nx
+
+      assert ~M[-1 2 3 4] == Nx.tensor([-1, 2, 3, 4])
```

Comment:
I have been thinking more about this and it seems to be impossible to specify a tensor that is a single row with multiple columns. Given the sigil is called `~M`, maybe this should always be a matrix? And we can introduce `~V` if we want to support vectors later on?

Reply:
Good point! Can I add `~V[-1 2 3 4]` as equivalent of `Nx.tensor([-1, 2, 3, 4])` now?

Reply:
Sure!